ui/src/features/Editor/components/OverlayUI/components/Collaboration/CollaborationCard.tsx

```diff
@@ -40,25 +48,40 @@ const CollaborationCard: React.FC<Props> = ({
           {userName.charAt(1)}
         </span>
       </div>
-      <div className="flex-grow">
+      <div className="flex flex-col">
         <span className="text-sm select-none dark:font-light">{userName}</span>
+        {time && (
+          <span className="text-sm opacity-55 dark:font-light">
+            {t("debugging (started {{time}})", {
+              time,
+            })}
+          </span>
+        )}
       </div>
-      {isHovered && !isSpotlighted && !self && (
+      {isHovered && onSpotlightUserSelect && !isSpotlighted && !self && (
         <IconButton
           className="h-8"
           tooltipText={t("Spotlight User")}
           icon={<TargetIcon size={14} />}
           onClick={() => onSpotlightUserSelect?.(clientId)}
         />
       )}
-      {isSpotlighted && !self && (
+      {isSpotlighted && onSpotlightUserDeselect && !self && (
         <IconButton
           className="h-8"
           tooltipText={t("Remove Spotlight")}
           icon={<ProhibitIcon size={14} />}
           onClick={() => onSpotlightUserDeselect?.()}
         />
       )}
+      {isHovered && onDebugRunJoin && !self && (
+        <IconButton
+          className="h-8"
+          tooltipText={t("View Debug Run")}
+          icon={<BinocularsIcon size={14} />}
+          onClick={() => onDebugRunJoin?.()}
+        />
+      )}
```

Comment:
You can do the same for the other IconButtons! 

```suggestion
      {isHovered && onDebugRunJoin && !self && (
        <IconButton
          className="h-8"
          tooltipText={t("View Debug Run")}
          icon={<BinocularsIcon size={14} />}
          onClick={onDebugRunJoin}
        />
      )}
```

Reply:
✅